github/ci/prow-deploy/files/jobs/kubevirt/kubevirt/kubevirt-periodics.yaml

```diff
@@ -1062,7 +1062,7 @@ periodics:
           - name: KUBEVIRT_QUARANTINE
             value: "true"
           - name: TARGET
-            value: k8s-1.17
+            value: k8s-1.20
```

Comment:
I think the name periodic-kubevirt-e2e-k8s-prev-prev-rook-ceph should also be updated.
What is current in this context that we use to compute prev-prev?

Reply:
Adjusted to not have prev-prev in the name, thanks.